src/skins/vector/css/matrix-react-sdk/structures/_MyGroups.scss

```diff
@@ -115,6 +115,10 @@ limitations under the License.
     cursor: pointer;
 }
 
+.mx_GroupTile_avatar {
+    cursor: -webkit-grab;
```

Comment:
![red-panda-445px](https://user-images.githubusercontent.com/986903/36251780-56b27320-123a-11e8-987e-ece640394427.jpg)
Don't forget about Firefox. Look at that little face.

Looks like Firefox supports the unprefixed version, so I'd just add that.